client/src/components/SearchBar.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import styled from 'styled-components';
+import {FlexRow, Container, Fonts} from './Constants.jsx';
+
+const Magnifier = styled.svg`
+display: flex;
+height: 16px;
+max-width: 50%;
+margin-right: 8px;
+stroke: currentcolor;
+stroke-width: 3;
+`;
+
+const Input = styled.input`
+cursor: text !important;
+border: none;
+outline: none;
+color: rgb(34, 34, 34);
+background-color: transparent;
+font-family: ${Fonts.family};
+font-size: ${Fonts.large};
+font-weight: ${Fonts.normal};
+line-height: 20px;
+flex: 1 1 0%;
+text-overflow: ellipsis;
+`;
+
+const SearchContainer = styled.div`
+display: flex;
+width: 100%;
+margin-left: -7vw;
+margin-top: -4vh;
+border: none;
+color: rgb(34, 34, 34);
+background-color: rgb(247, 247, 247);
+align-items: center;
+padding: 12px 12px 12px 16px;
+border-radius: 100px;
+box-shadow: rgb(176, 176, 176) 0px 0px 0px 1px inset;
+`;
+
+class SearchBar extends React.Component {
+  constructor({callback}) {
+    super();
+    this.search = callback;
```

Comment:
could be a stateless component! no state

Reply:
Added state to my search bar since making this branch but nice suggestion